docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -25,11 +25,16 @@ needs.
 
 See the :doc:`kafka-tutorial`.
 
-.. note::
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a Kafka
+data source are:
 
-    The minimum supported Kafka broker version is 0.10.0.
-    The connector is tested against Confluent Kafka versions 5.2.1 and 5.5.2,
-    but any intermediate or newer versions are expected to work.
+* Kafka broker version is 0.10.0 or higher, testing performed against Confluent
```

Comment:
"testing performed" by who? I know, but the reader in a hurry won't know. The original Note was more explicit. How about:

"...0.10.0 or higher. Trino testing is performed against Confluent Kafka versions ..."

Reply:
I am torn to be honest since it really can only be testing by the Trino project and as such not necessary to call out ..

Reply:
The testing note isn't important for requirements. I'd leave it out, it's not necessary for the section.

Reply:
removed

Reply:
Just putting this here. I loved how Debezium had a matrix. https://debezium.io/releases/

Ours can be simpler because the only "supported" Trino version is the latest.
It might be costly from a maintainence perspective though unless there is a process ensuring it gets updated when changes are merged.

Reply:
I think for starters we have to get the info we have now into all the connectors and become good at updating that. JDBC driver version might be a good detail to expose but I am not sure about that.

Once we have all connectors and such making a matrix in addition might make sense too